presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduClientSession.java

```diff
@@ -500,20 +500,31 @@ else if (domain.isSingleValue()) {
                     }
                     else if (valueSet instanceof SortedRangeSet) {
                         Ranges ranges = ((SortedRangeSet) valueSet).getRanges();
-                        Range span = ranges.getSpan();
-                        Marker low = span.getLow();
-                        if (!low.isLowerUnbounded()) {
-                            KuduPredicate.ComparisonOp op = (low.getBound() == Marker.Bound.ABOVE)
-                                    ? KuduPredicate.ComparisonOp.GREATER : KuduPredicate.ComparisonOp.GREATER_EQUAL;
-                            KuduPredicate predicate = createComparisonPredicate(columnSchema, op, low.getValue());
+                        List<Range> rangeList = ranges.getOrderedRanges();
+                        if (rangeList.stream().allMatch(Range::isSingleValue)) {
+                            io.prestosql.spi.type.Type type = TypeHelper.fromKuduColumn(columnSchema);
```

Comment:
can you use import?

Reply:
org.apache.kudu.Type is imported, So qualified name should be used here.